channeldb/payment_control_test.go

```diff
@@ -207,10 +216,9 @@ func TestPaymentControlSwitchDoubleSend(t *testing.T) {
 	// payment hash, should result in error indicating that payment has
 	// already been sent.
 	err = pControl.InitPayment(info.PaymentIdentifier, info)
-	if err != ErrPaymentInFlight {
-		t.Fatalf("payment control wrong behaviour: " +
-			"double sending must trigger ErrPaymentInFlight error")
-	}
+	require.Equal(t, err, ErrPaymentExits, "payment control wrong "+
```

Comment:
nit: actual and expected values are swapped.

Reply:
ah my bad, fixed!